frontend/src/components/buttons/DeleteButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from "primereact/button";
+
+interface DeleteButtonProps {
+  isEnabled: boolean;
+  onClick: () => void;
+}
+
+export default function DeleteButton(props: DeleteButtonProps) {
+  return (
+    <div className="flex col-1">
```

Comment:
we don't want to factor out <div className="flex col-1"> this is page specific code that creates margins and padding for each JSX element.